examples/WiFiAdvancedChatServer/WiFiAdvancedChatServer.ino

```diff
@@ -0,0 +1,108 @@
+/*
+ Advanced WiFi Chat Server
+
+ A more advanced server that distributes any incoming messages
+ to all connected clients but the client the message comes from.
+ To use, telnet to your device's IP address and type.
+ You can see the client's input in the serial monitor as well.
+
+ Circuit:
+  * Board with NINA module (Arduino MKR WiFi 1010, MKR VIDOR 4000 and UNO WiFi Rev.2)
+
```

Comment:
@JAndrassy Can you please add a note here that this sketch requires firmware 1.5.0 or higher ?

Reply:
Actually never mind, I see that you check for the latest firmware version.

Reply:
If this example is intended to work only with nina-fw >= 1.5.0 i think it would be better to stop execution, if the condition is not met, like we do for the WiFi module check.

```
  // check for the WiFi module:
  if (WiFi.status() == WL_NO_MODULE) {
    Serial.println("Communication with WiFi module failed!");
    // don't continue
    while (true);
  }
```